app/views/oaisys/pmh/list_sets.xml.builder

```diff
@@ -1,6 +1,6 @@
 xml.push_deferred_attribute('xmlns:dc': 'http://purl.org/dc/elements/1.1/',
                             'xmlns:oai-id': 'http://www.openarchives.org/OAI/2.0/oai-identifier',
-                            'xmlns': 'http://www.openarchives.org/OAI/2.0/',
```

Comment:
It was because of this cop: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/SymbolConversion

They should have an identical result.  But maybe we'd prefer `EnforcedStyle: consistent` so there isn't just the one namespace without a colon that's different.

Reply:
I asked rubocop to enforce the consistent style instead.